### PR TITLE
2022 cuts

### PR DESCRIPTION
**Pull request by nrc**
r? @rylev as discussed (I think)

### CODE REVIEW

File: surveys/2022-annual-survey/questions.md [hunk 1]
```diff
@@ -26,41 +26,12 @@ Type: select one
...
->
-> Useful in understanding why non-users contribute;
-
 > **SURVEY FLOW**
 >
 > Skip to `## Your opinions about Rust` section
 
 ## For non-Rust users
 
-### As you have indicated that you have not used Rust, what prompted you to participate in this survey?
```
Comment (rylev) [inline surveys/2022-annual-survey/questions.md:50]: Perhaps we want to replace these with a TODO just so we're sure we don't forget that we should either make new questions here or delete these sections entirely.

File: surveys/2022-annual-survey/questions.md [hunk 2]
```diff
@@ -669,12 +596,6 @@ Aspects:
 - Disk space usage (e.g., the size of target folder)
 - Bugs in the compiler (i.e., ICEs a.k.a. internal compiler errors, miscompilations, etc.)
 - Compiler error messages
-- IDE experience
-- Debugging experience
-- Available tools and support
-- Async programming
-- GUI development
-- Rust language and standard library documentation
```
Comment (rylev) [inline surveys/2022-annual-survey/questions.md:677]: Thinking more about it, I'm not sure about this...

**Review by rylev — CHANGES_REQUESTED** [review]
I got cold feet on removing some of the "aspects of Rust" that we ask respondents to answer... Otherwise, looks great (minus a small nit).

File: surveys/2022-annual-survey/questions.md (hunk 2 above)
Comment (nrc) [reply surveys/2022-annual-survey/questions.md:677]: Note to self (from Wesley): could we get respondents to rank these in priority order, to discover how people would feel about sacrificing, e.g., disk space for compile times?

File: surveys/2022-annual-survey/questions.md (hunk 1 above)
Comment (nrc) [reply surveys/2022-annual-survey/questions.md:50]: I have the TODOs in a separate branch

File: surveys/2022-annual-survey/questions.md (hunk 2 above)
Comment (rylev) [reply surveys/2022-annual-survey/questions.md:677]: I think having all of these rated would be useful.

Comment (rylev) [reply surveys/2022-annual-survey/questions.md:677]: @nrc can you add these back, and then we can discuss separately whether to cut them?

**Comment by nrc** [comment]
Tools stuff restored

**Review by rylev — APPROVED** [review]

**Pull request merged**